src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,762 @@
+# Non Fungible Tokens (NFTs)
+
+The AtomicDEX API supports [ERC1155](https://www.nftstandards.wtf/Standards/ERC1155+Multi+token) and [ERC721](https://www.nftstandards.wtf/Standards/ERC721+Non+Fungible+Standard) NFTs via the [Moralis API](https://docs.moralis.io/) on the Avalanche (AVAX), BNB Smart Chain (BNB), Ethereum (ETH), Fantom (FTM), Polygon (MATIC) networks.
+
+
+
+## NFT Information Methods
+
+- Get a list of your tokens with [get_nft_list](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-list)
+- Get a list of token transfers with [get_nft_transfers](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-transfers)
+- Get token metadata with [get_nft_metadata](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-metadata)
+
+
+## NFT Transaction Methods
+
+- Withdraw ERC721 tokens with [withdraw_nft](/atomicdex/api/v20-dev/non-fungible-tokens/withdraw-nft#erc721)
+- Withdraw ERC1155 tokens with [withdraw_nft](/atomicdex/api/v20-dev/non-fungible-tokens/withdraw-nft#erc1155)
+
+
+## Get a list of NFTs {{label : 'get_nft_list', tag : 'API-v2'}}
+
+Returns a list of the NFTs owned by the user, shown in descending order of the `block_number` value (the block height when the amount or owner changed). If the request is for NFTs on more than one chain, this means that the order may not be chronological.
```

Comment:
`the block height when the amount or owner changed`

the note: if we are talking about amount, it can be changed if NFT has erc1155 standard type, but not always. 
when user receive more erc1155 tokens, then `block_number` rises, but when user transfer not all erc1155 tokens, usually `block_number` stays the same, bcz[ `safeTransferFrom` function](https://eips.ethereum.org/EIPS/eip-1155#specification) does not affect `block_number` changes. well potentially NFT creator can implement this functionality, but it increases fee cost. so usually from my experience `block_number` stayed the same after transferring some erc1155 amount.

Reply:
updated in https://github.com/KomodoPlatform/komodo-docs-mdx/pull/42/commits/0e74d8b2e61dfa43e7db9136a05ece3b73ad6748